receiver/hostmetricsreceiver/hostmetrics_receiver_test.go

```diff
@@ -72,7 +72,7 @@ var resourceMetrics = []string{
 }
 
 var systemSpecificMetrics = map[string][]string{
-	"linux":   {"system.disk.merged", "system.filesystem.inodes.usage", "system.paging.faults", "system.processes.created", "system.processes.count"},
+	"linux":   {"system.disk.merged", "system.disk.weighted_io_time", "system.filesystem.inodes.usage", "system.paging.faults", "system.processes.created", "system.processes.count"},
```

Comment:
do we have specs for this? If no we should at least have an issue to track this in the specs.

Reply:
@aabmass This metric is specific to Linux, although `system.disk.merged` is as well.

Should these metrics actually be prefixed with `system.linux` as per the guidance [here](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/metrics/semantic_conventions/system-metrics.md#systemos---os-specific-system-metrics) or should we make an exception for these two and add `system.disk.weighted_io_time` to the specs as well?